README.md

```diff
@@ -10,3 +10,67 @@ The Mattermost operator is in alpha, data loss might occur.
 Mattermost is a scalale, open source collaboration tool. It's written in Golang and React.
 
 The idea of this project is to build the Kubernetes Operator for Mattermost to mak more easy for Kubernetes users deploy and manage your Mattermost instance.
+
+
+## Getting Started
+
+### Prerequisites
+
+- Kubernetes cluster 1.8.0+.
+- `kubectl` configured to refer to relevant Kubernetes cluster.
+
+### Create Operator and related resources
+
+To start Mattermost-Operator, we need to install the dependencies first.
```

Comment:
Suggest eliminating this line:
```suggestion
```

and removing the `MySQL-Operator` and `Mattermost-Operator` headers below, to something like:
```
Install the mysql-operator for use by the mattermost-operator:
<kubectl commands>

Install the mattermost-operator:
<kubectl commands>
```

The fact that these manifests are in the `docs` folders does not -- IMO -- need to be called out explicitly, given they are in the commands being executed.